Makefile

```diff
@@ -493,3 +493,24 @@ install-kvm-driver: out/docker-machine-driver-kvm2
 release-kvm-driver: install-kvm-driver checksum
 	gsutil cp $(GOBIN)/docker-machine-driver-kvm2 gs://minikube/drivers/kvm/$(VERSION)/
 	gsutil cp $(GOBIN)/docker-machine-driver-kvm2.sha256 gs://minikube/drivers/kvm/$(VERSION)/
+
+site/themes/docsy/assets/vendor/bootstrap/package.js:
+	git submodule update -f --init --recursive
+
+# hugo for generating site previews
+.ONESHELL:
+out/hugo/hugo:
```

Comment:
maybe add a check to this, only if hugo is not in the path ?

Reply:
I thought about this, but didn't want to get into checking if it was the same version and compiled with the same options. This leaves us room to pin to a specific version later.